speech/check_dataVals.m

```diff
@@ -29,17 +31,32 @@
 if nargin < 3 || isempty(buffertype), buffertype = 'signalIn'; end
 if nargin < 5, folderSuffix = []; end
 if nargin < 6, errorParams = []; end
+if nargin < 7, sigs2plot = {'f1','f2'}; end
+if ~iscell(sigs2plot)
+    error('sigs2plot must be a cell array. if you input a string, try adding braces.')
+end
```

Comment:
```suggestion
if ~iscell(sigs2plot)
    if ischar(sigs2plot)
        sigs2plot = {sigs2plot};
    else
        error('sigs2plot must be a cell array (e.g. {'f1','f2'}) or a single character array (e.g. 'f1').')
    end
end
```

Reply:
Fixed in b29105ca281f505c83339ce68d5f78c19149b596